### DISCUSSION TITLE
Detect value change in a field

### DISCUSSION THREAD

**Discussion by buuum** [directus/directus#6779] [Archive]
Hello,

I have in the user collection two boolean fields: smoker and hasChildren

How do I trigger events when these values change?

From create or update hook I have no way to check if the value has changed, right?

Should I create a custom endpoint to update those data? 

Thanks

**Comment by rijkvanzanten** [comment]
> From create or update hook I have no way to check if the value has changed, right?

You could fetch the currently saved item in the `items.update.before` hook, and compare it to the submitted payload 👍🏻
